helix-term/src/commands.rs

```diff
@@ -2641,6 +2641,56 @@ pub mod cmd {
         Ok(())
     }
 
+    fn sort(
+        cx: &mut compositor::Context,
+        args: &[Cow<str>],
+        _event: PromptEvent,
+    ) -> anyhow::Result<()> {
+        sort_impl(cx, args, false)
+    }
+
+    fn sort_reverse(
+        cx: &mut compositor::Context,
+        args: &[Cow<str>],
+        _event: PromptEvent,
+    ) -> anyhow::Result<()> {
+        sort_impl(cx, args, true)
+    }
+
+    fn sort_impl(
+        cx: &mut compositor::Context,
+        _args: &[Cow<str>],
+        reverse: bool,
+    ) -> anyhow::Result<()> {
+        let (view, doc) = current!(cx.editor);
+        let text = doc.text().slice(..);
+
+        let selection = doc.selection(view.id);
+
+        let mut fragments: Vec<_> = selection
+            .fragments(text)
+            .map(|fragment| Tendril::from_slice(&fragment))
+            .collect();
+
+        fragments.sort_by(match reverse {
+            true => |a: &Tendril, b: &Tendril| b.cmp(a),
+            false => |a: &Tendril, b: &Tendril| a.cmp(b),
+        });
```

Comment:
https://doc.rust-lang.org/std/cmp/struct.Reverse.html would make it clearer when you're reverse sorting (we already import it in the file)

Reply:
How do we use it though? `Reverse` is a struct, so if we try to use it in a match or if/else, it complains that the types are incompatible. It looks like it was meant to use with `sort_by_key` and friends where there is only one thing to reference.